modules/dhcp_common/isc/configuration_parser.rb

```diff
@@ -209,10 +209,11 @@ def server_duid
           Rsec::Fail.reset
           keyword = word('server-duid').fail 'keyword_server_duid'
           anything = /[^;,{}\s]+/.r
+          literal = /"([^"]|\")*"/.r
```

Comment:
It looks like ISC configuration parser treats all strings like that, isn't cleaner to use this in `LITERAL` constant?

https://github.com/42wim/isc-dhcp/blob/f54a146c7fe88889d60f0c1aa8e6f04707f95223/common/conflex.c#L512